src/java.base/share/classes/sun/security/util/TlsChannelBinding.java

```diff
@@ -78,25 +70,6 @@ public String getName() {
         }
     }
 
-    /**
-     * Parse value of "com.sun.jndi.ldap.tls.cbtype" property
-     * @param  cbType
-     * @return TLS Channel Binding type or null if
-     *         "com.sun.jndi.ldap.tls.cbtype" property has not been set.
-     * @throws ChannelBindingException
-     */
-    public static TlsChannelBindingType parseType(String cbType) throws ChannelBindingException {
-        if (cbType != null) {
-            if (cbType.equals(TlsChannelBindingType.TLS_SERVER_END_POINT.getName())) {
-                return TlsChannelBindingType.TLS_SERVER_END_POINT;
-            } else {
-                throw new ChannelBindingException("Illegal value for " +
-                        CHANNEL_BINDING_TYPE + " property.");
-            }
-        }
-        return null;
-    }
-
```

Comment:
I think this method should stay here. Suppose one day the CBT type is configurable for HTTPS we'll have to get it back. Of course we will need to update the message to avoid talking about LDAP.

Reply:
The 2 strings should be on the LDAP side. This method does not really depend on the strings except for mentioning one in the exception message. We can just rewrite it into `"Illegal channel binding type: " + cbType`.